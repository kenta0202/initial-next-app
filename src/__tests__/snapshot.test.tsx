/* eslint-disable @typescript-eslint/no-unsafe-call */
/* eslint-disable @typescript-eslint/no-unsafe-member-access */
/* eslint-disable @typescript-eslint/no-unsafe-assignment */
import TestingPage from "pages/sample/other/testing"
import renderer from "react-test-renderer"

it("renders TestingPage unchanged", () => {
  const tree = renderer.create(<TestingPage />).toJSON()
  expect(tree).toMatchSnapshot()
})
