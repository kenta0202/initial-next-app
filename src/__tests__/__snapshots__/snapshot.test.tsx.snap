// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TestingPage unchanged 1`] = `
[
  <main>
    <h1>
      Hello
    </h1>
    <p>
      Hello
    </p>
    <p
      data-testid="test"
    >
      Test
    </p>
  </main>,
  <div
    style={
      {
        "margin": "2em",
      }
    }
  >
    <h1>
      電源
       
      OFF
    </h1>
    <button
      disabled={false}
      onClick={[Function]}
      onPointerDown={[Function]}
    >
      ON
    </button>
    <button
      disabled={true}
      onClick={[Function]}
      onPointerDown={[Function]}
    >
      OFF
    </button>
  </div>,
]
`;
